### PR TITLE
Redesigned V0 Thumb Nuts

### PR DESCRIPTION
**Pull request by ardichoke**
#### Which mods/configurations/profiles are added by this PR?
Voron V0 longer thumb nuts

#### How was it tested? 
Printed a few different iterations, installed them on my bed and made sure they worked

#### Any background context you want to provide?
I found the thumb nuts to be very uncomfortable to use, and too short to access easily when using the kirigami bed mount. This redesign has knurling that I find more comfortable on my digits and is taller to make them easier to access.

#### Screenshots (if appropriate)
See images in the PR

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Discord to have your mod moved to this folder.
 ## Table structure
 
 Update the following table with the information about your mod:
+
```
Comment (andrepl) [inline printer_mods/README.md:15]: Looks like there are several unrelated modifications to this README, please limit changes here to *only* your entry

**Review by andrepl — COMMENTED** [review]
file and folder names shouldn't contain spaces.  `CAD/Bed Thumb Nut.step` should be renamed.

File: printer_mods/README.md (hunk 1 above)
Comment (MapleLeafMakers) [inline printer_mods/README.md:15]: Please limit changes in this README to only your own entry. there are several unrelated changes here.

Comment (ardichoke) [reply printer_mods/README.md]: Sorry, my editor is set up to automatically fix violations of the markdown style guide. I've fixed this.

**Comment by ardichoke** [comment]
> file and folder names shouldn't contain spaces. `CAD/Bed Thumb Nut.step` should be renamed.

Addressed this

**Review by cbc02009 — APPROVED** [review]
